extensions/amp-experiment/1.0/mutation/attribute-mutation-default-style.js

```diff
@@ -27,42 +37,82 @@ export class AttributeMutationDefaultStyle {
    * @param {!Array<Element>} elements
    */
   constructor(mutationRecord, elements) {
+    /** @private {!JsonObject} */
     this.mutationRecord_ = mutationRecord;
+
+    /** @private {!Array<Element>} */
```

Comment:
is the Element nullable?

Reply:
The element are from `querySelectorAll`, so they are not nullable.

Reply:
so it can be `{!Array<!Element>}`

Reply:
done